packages/nextjs/components/scaffold-eth/FaucetButton.tsx

```diff
@@ -21,7 +22,10 @@ const localWalletClient = createWalletClient({
  */
 export const FaucetButton = () => {
   const { address } = useAccount();
-  const { balance } = useAccountBalance(address);
+
+  const { data: balance } = useBalance({
```

Comment:
Since `balance` here is object, the `className={balance ? 'ml-1' : '...'}` at line 65 will always be true and we will never show the tooltip

Lets maybe use a variable to track it maybe something like : 

```ts
const isBalanceZero = balance && balance.value === 0n;
```

So if `isBalanceZero` is true we show the tooltip

Reply:
Here is the context I was missing. Makes sense.

The suggested change is now reflected in this commit:
[86841363057bdf0e5d1f2091a43e9ef3dc9e2699](https://github.com/scaffold-eth/scaffold-eth-2/pull/788/commits/86841363057bdf0e5d1f2091a43e9ef3dc9e2699)